mathjax3-ts/input/tex/ParseUtil.ts

```diff
@@ -488,6 +488,30 @@ namespace ParseUtil {
     return (font ? {mathvariant: font} : {});
   };
 
+
+  /**
+   * Splits a package option list of the form [x=y,z=1] into an attribute list
+   * of the form {x: y, z: 1}.
+   * @param {string} attrib The attributes of the package.
+   * @param {{[key: string]: number}?} allowed A list of allowed options.
+   * @return {EnvList} The attribute list.
+   */
+  export function splitPackageOptions(attrib: string, allowed: {[key: string]: number} = {}): EnvList {
+    let def: EnvList = {};
+    if (attrib !== '') {
+      const attr = attrib.replace(/ /g, '').split(/,/);
```

Comment:
I had occasion recently to need a function like this in a v2 extension, but the values that can be assigned could contain braced groups (so that you could include commas and equal signs, for example).  It might be good to handle that setup (which unfortunately means you can't just use splitting).  Alternatively, we could do another function when that is needed.

Reply:
I've now implemented the full functionality of the LaTeX `keyval` function (https://www.ctan.org/pkg/keyval), with the exception that we remove arbitrary pairs of outer braces, while `keyval` only removes two pairs. But that should not make a difference.